content/recipes/authoring/kata-snippets/preloaded.md

```diff
@@ -7,7 +7,7 @@ next: /recipes/authoring/kata-snippets/reference-solution/
 
 # Writing Preloaded Code
 
-Reference Solution snippet, created by kata athor or translator, can be used to reduce boilerplate code, but when used incorrectly, it can cause lots of problems. To help keep kata quality good, it should conform to a set of guidelines presented below.
+Preloaded Code snippet, created by kata athor or translator, can be used to reduce boilerplate code, but when used incorrectly, it can cause lots of problems. To help keep kata quality good, it should conform to a set of guidelines presented below.
```

Comment:
```suggestion
The preloaded code snippet, created by the kata author or translator, can be used to reduce boilerplate code, but when used incorrectly, it can cause lots of problems. To ensure the quality of the kata, it should conform to a set of guidelines presented below.
```